followed-accounts.js

```diff
@@ -0,0 +1,133 @@
+import { db } from './dbInstance.js'
+
+class FollowedActorsList extends HTMLElement {
+  constructor () {
+    super()
+  }
+
+  connectedCallback () {
+    this.renderFollowedActors()
+
+    window.addEventListener(
+      'exportFollowed',
+      FollowedActorsList.exportFollowedList
+    )
+
+    window.addEventListener('importFollowed', (e) => {
+      FollowedActorsList.importFollowedList(e.detail.file)
+    })
+
+    // Listen for the custom event to refresh the list and count
+    window.addEventListener('followedActorsUpdated', async () => {
```

Comment:
We should turn the DB class into extending EventTarget and emit events on there instead. It keeps the logic closer together and will allow for better testing and multi-db use cases in the future